<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reviewappClass</class>
 <widget class="QMainWindow" name="reviewappClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reviewapp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="filterTextBox">
      <property name="placeholderText">
       <string>搜索问题...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>记录列表:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="recordListView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="editBtn">
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBtn">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="favBtn">
            <property name="text">
             <string>收藏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewAnswerBtn">
            <property name="text">
             <string>查看答案</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>问题:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>答案:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>难度评分:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="hardBtn">
            <property name="text">
             <string>困难</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="normalBtn">
            <property name="text">
             <string>一般</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="easyBtn">
            <property name="text">
             <string>简单</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReview"/>
   <addaction name="actionAll"/>
   <addaction name="actionFav"/>
   <addaction name="actionTest"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionTheme"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReview">
   <property name="text">
    <string>复习</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>所有</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFav">
   <property name="text">
    <string>收藏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>测试</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>主题</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="reviewapp.qrc"/>
 </resources>
 <connections/>
</ui>
